File: packages/ios/Sources/Components/POICard/POICard.figma.swift
import SwiftUI
import Figma

struct KozmosPOICardConnect: FigmaConnect {
    let component = KozmosPOICard<EmptyView, Text, EmptyView>.self
    let figmaNodeUrl = "https://figma.com/design/Yj4O8p6Y9h2Sa9zJVoAiVY?node-id=1340-6879"

    @FigmaString("Title Text")
    var title: String = "Kozmos Cafe"

    @FigmaString("Subtitle Text")
    var category: String = "Cafe"

    @FigmaString("Description Text")
    var descriptionText: String = "Speciality coffee beside the north atrium."

    var body: some View {
        KozmosPOICard(
            title: self.title,
            category: self.category,
            imageContent: { EmptyView() },
            descriptionContent: { Text(self.descriptionText) },
            actionsContent: { EmptyView() },
            onClose: {}
        )
    }
}

File: packages/android/src/main/java/com/kozmos/components/POICard/POICard.figma.kt
package com.kozmos.components.poicard

import androidx.compose.runtime.Composable
import com.figma.code.connect.Figma
import com.figma.code.connect.FigmaConnect
import com.figma.code.connect.FigmaProperty
import com.figma.code.connect.FigmaType

@FigmaConnect("https://figma.com/design/Yj4O8p6Y9h2Sa9zJVoAiVY?node-id=1340-6879")
class KozmosPOICardConnect {
    @FigmaProperty(FigmaType.Text, "Title Text")
    val title: String = "Kozmos Cafe"

    @FigmaProperty(FigmaType.Text, "Subtitle Text")
    val category: String = "Cafe"

    @Composable
    fun ComponentExample() {
        KozmosPOICard(title = title, category = category, onClose = {})
    }
}
